temurin-install-testing/README.md

```diff
@@ -0,0 +1,48 @@
+# temurin-testing
+
+This Terraform configuration installs Temurin on a matrix of GCE machine types and OS boot images.
+The tests are split into three batches to prevent hitting standard compute quotas.
```

Comment:
I expected some explanation of three batches to follow. But I see 4 item bullet point.